tests/unit/ml_handlers/data/house_sales.csv

```diff
@@ -0,0 +1,348 @@
+saledate,ma,type,bedrooms

```

Comment:
@ZoranPandovski thoughts on this? My impression was we already had some datasets in the tests folder, but it seems like I was mistaken. Any preference on where this should go?